build.gradle

```diff
@@ -205,6 +205,11 @@ def versions = [
 
 ext["rest-assured.version"] = '4.3.0'
 
+configurations.all {
+  exclude group: 'org.springframework.security', module: 'spring-security-rsa'
+  exclude group: 'org.springframework.boot', module: 'spring-boot-starter-security'
+}
```

Comment:
Please revert the other changes in this PR except build.gradle. There is another PR you raised for letter status change https://github.com/hmcts/send-letter-service/pull/1559

Reply:
Done.